content/technology/architecture/components/application/construction/altinn-apps/app/app-backend/process/_index.md

```diff
@@ -24,11 +24,6 @@ BPMN reader is the component that parses BPMN process in apps.
 
 [See github for source code](https://github.com/Altinn/altinn-studio/blob/master/src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Process/BpmnReader.cs).
 
-## Process Step App Frontend
```

Comment:
I'm not really sure what this is about, but the code this references doesnt exist anymore, so I figured this could be removed. Or should it point to somewhere else?

@TheTechArch @lorang92 Do any of you know?

Reply:
This has been renamed to `ProcessWrapper` - I guess we can point there. Not really sure how much value it adds to have a section on docs on that. Thoughts?

Reply:
Hm, all of the links on this page seems to be dead actually...